packages/core-blockchain/lib/utils/tick-sync-tracker.js

```diff
@@ -12,22 +14,22 @@ module.exports = async block => {
       start: new Date().getTime(),
       networkHeight: container.resolvePlugin('p2p').getMonitor().getNetworkHeight(),
       downloadedBlocks: height,
-      blockPerSecond: 0,
+      blockPerMs: 0,
       percent: 0,
       secondsLeft: 0
     }
   }
 
   tracker.downloadedBlocks += 400
   tracker.percent = (tracker.downloadedBlocks * 100) / tracker.networkHeight
-  tracker.blockPerSecond = ((new Date().getTime()) - tracker.start) / 1000 / tracker.downloadedBlocks
-  tracker.secondsLeft = (tracker.networkHeight - tracker.downloadedBlocks) / tracker.blockPerSecond
+  tracker.blockPerMs = ((new Date().getTime()) - tracker.start) / tracker.downloadedBlocks
+  tracker.msLeft = (tracker.networkHeight - tracker.downloadedBlocks) / tracker.blockPerMs
 
   if (tracker.percent < 100) {
     const downloadedBlocks = tracker.downloadedBlocks.toLocaleString()
     const networkHeight = tracker.networkHeight.toLocaleString()
 
-    logger.printTracker('Fast Sync', tracker.percent, 100, `(${downloadedBlocks} of ${networkHeight} blocks)`)
+    logger.printTracker('Fast Sync', tracker.percent, 100, `(${downloadedBlocks} of ${networkHeight} - ${prettyMs(tracker.msLeft, {secDecimalDigits: 0})})`)
```

Comment:
The time seems to be quite a bit off. It starts at 9 hours and then jumps to 50 seconds.